study-builder/studies/prion/emails/contact_medical_reminder.en.html

```diff
@@ -0,0 +1,83 @@
+
+                            <tr  style="height: 0">
+                                <td style="border: 0 solid rgb(0, 0, 0); vertical-align: top; background-color: rgb(255, 255, 255); padding: 20px 0 0;">
+                                    <p dir="ltr" style="margin-top: 0; margin-bottom: 0; line-height: 1.875;"><span
+                                            style="font-family: Arial, sans-serif; font-size: 16px; color: rgb(102, 102, 102); white-space: pre-wrap;background-color:
+                                            rgb(255, 255, 255);">It has been over a year since information for [ParticipantFirstName ParticipantLastName] on PrionRegistry.org was updated. You are listed as a contact for this account. When you have a chance, please <a href="PRIONURL/login-landing/login">log in</a> and update the Medical Questionnaire so that we can maintain an accurate count of our participants and so that we can reach you to let you know about any opportunities to participate in research.
+</span></p>
+                                </td>
+                            </tr>
+
+                            <tr style="height: 0">
+                                <td style="border: 0 solid rgb(0, 0, 0); vertical-align: top; background-color: rgb(255, 255, 255); padding: 20px 0 0;">
+                                    <p dir="ltr" style="margin-top: 0; margin-bottom: 0; line-height: 1.875;"><span
+                                            style="font-family: Arial, sans-serif; font-size: 16px; color: rgb(102, 102, 102); white-space: pre-wrap;background-color: rgb(255, 255, 255);">Thank you for being a part of our community and our drive for a cure.
+
+</span></p>
+                                </td>
+                            </tr>
+
+                            <tr  style="height: 0">
+                                <td style="border: 0 solid rgb(0, 0, 0); vertical-align: top; background-color: rgb(255, 255, 255); padding: 20px 0 0;">
+                                    <p dir="ltr" style="margin-top: 0; margin-bottom: 0; line-height: 1.875;"><span
+                                            style="font-family: Arial, sans-serif; font-size: 16px; color: rgb(102, 102, 102); background-color: rgb(255, 255, 255);">Best,<br/>
+Prion Registry Administrators
+</span></p>
+                                </td>
+                            </tr>
+
+                            <tr  style="height: 0">
+                                <td style="border: 0 solid rgb(0, 0, 0); vertical-align: top; background-color: rgb(255, 255, 255); padding: 20px 0 0;">
+                                    <p dir="ltr" style="margin-top: 0; margin-bottom: 0; line-height: 1.875;"><span
+                                            style="font-family: Arial, sans-serif; font-size: 16px; color: rgb(102, 102, 102); background-color: rgb(255, 255, 255);"><a href="[unsubscribe]">unsubscribe</a> | <a href="PRIONURL/login-landing/login">reset password</a>
```

Comment:
Is the idea that study staff will manually replace `PRIONURL` and stuff in `[]` brackets themselves before sending in SendGrid?

Reply:
SendGrid automatically replaces `[unsubscribe]` with the URL of their unsubscribe page.  When I added this to the dev SendGrid account, I substituted the appropriate URL for Prion as `PRIONURL`, but eventually this will need to be set up in some sort of production SendGrid account so I didn't want to specify which URL to use.